### ISSUE TITLE
ISSN 2340-0080 | Estudios penales y criminologicos (PKP) | Rezensionserkennung 

### ISSUE THREAD

**Issue by kyushutea** [ubtue/DatenProbleme#2080]
#### URL
https://revistas.usc.gal/index.php/epc/issue/view/550
  
#### Import-Translator
  Einzelimport:
ubtue_PKP Catalog Systems.js

  
### Problembeschreibung
- [ ] Halbautomatisch: 

Die drei Rezensionen unter der Rubrik "Recensiones" werden im Einzelimport im **halbautomatischen Verfahren** erkannt. Mehrfachimport mit PKP wird angeboten, funktioniert, aber nicht richtig.

- [x] Vollautomatisch: 

Im **vollautomatischen Verfahren** werden Rezensionen nicht erkannt.

**Comment by jriedl** [comment]
Auf nu.

**Comment by kyushutea** [comment]
Danke, funktioniert auf nu und ub28.

**Comment by kyushutea** [comment]
Siehe Klärungsbedarf: https://github.com/ubtue/DatenProbleme/issues/2084#issuecomment-2304598178

Beim Einzelimport mit ubtue_Open Journal Systems Standard.js wird nicht die Rezension erkannt.
Beim Einzelimport mit ubtue_PKP Catalog Systems.js wird die Rezension erkannt.

Getestet mit dem Link: https://revistas.usc.gal/index.php/epc/article/view/8569/12915

**Comment by mara-spiess** [comment]
@kyushutea Die Priorität hier wäre, dass der ubtue_PKP Translator auch im Mehrfach Import funktioniert, sehe ich das richtig?

**Comment by kyushutea** [comment]
@mara-spiess Richtig, der Mehrfachimport mit PKP wird angeboten, funktioniert jedoch nicht.

**Comment by mara-spiess** [comment]
Der Mehrfachimport sollte nun hoffentlich richtig funktionieren :)

**Comment by kyushutea** [comment]
Der Mehrfachimport funktioniert. Vielen Dank!
